benchmarks/src/main/scala/zio/chunks/ChunkAddBenchmark.scala

```diff
@@ -0,0 +1,50 @@
+package zio.chunks
+
+import java.util.concurrent.TimeUnit
+
+import org.openjdk.jmh.annotations._
+
+import zio.Chunk
+
+@State(Scope.Thread)
+@BenchmarkMode(Array(Mode.Throughput))
+@OutputTimeUnit(TimeUnit.SECONDS)
+class ChunkAddBenchmarks {
+
+  var chunk: Chunk[Int]   = _
+  var vector: Vector[Int] = _
+
+  @Param(Array("10000"))
+  var size: Int = _
+
+  @Setup(Level.Trial)
+  def setup() =
+    chunk = Chunk(1)
+  vector = Vector(1)
```

Comment:
You can even drop the setup and initialize them above.

Reply:
Done.